dist/obs-server.spec

```diff
@@ -492,9 +538,20 @@ touch /srv/www/obs/api/last_deploy || true
 /etc/init.d/obssrcserver
 /etc/init.d/obswarden
 /etc/init.d/obsdodup
-%{_unitdir}/obsdeltastore.service
 /etc/init.d/obsservicedispatch
 /etc/init.d/obssigner
+%{_unitdir}/obsdeltastore.service
```

Comment:
this backend stuff, must be in obs-server

Reply:
Leaving it there.